rasa_core/policies/embedding_policy.py

```diff
@@ -481,11 +480,15 @@ def _create_rnn_cell(self):
                 out_layer_size=embed_layer_size
         )
 
+    @staticmethod
+    def num_mem_units(memory):
+        return memory.shape[-1].value
+
     @staticmethod
     def _create_attn_mech(memory, real_length):
-        num_mem_units = memory.shape[-1].value
         attn_mech = tf.contrib.seq2seq.BahdanauAttention(
-                num_units=num_mem_units, memory=memory,
+                num_units=EmbeddingPolicy.num_mem_units(memory),
```

Comment:
this can be `self.num_mem_units(memory)` I guess, there is no need for this method to be static